openlane/scripts/odbpy/defutil.py

```diff
@@ -278,7 +293,10 @@ def relocate_pins(db, input_lefs, template_def, permissive):
         name = output_bterm.getName()
         output_bpins = output_bterm.getBPins()
 
-        if name not in template_bterm_locations or name not in all_bterm_names:
+        if name not in template_bterm_locations:
+            continue
+
+        if (name not in all_bterm_names) and not copy_def_power:
```

Comment:
This line specifically is what causes the regression when it's not true by default, I think- I'll investigate further

Reply:
See above. The `template_bterm_names` I overwrite is wrong, it's a mistage and should be removed.
This is causing a regression because before power pins would be ignored and not cause mismatch.  And now they do cause a mismatch.  I'm rechecking which that chunk removed now ...

Reply:
Ah…

Reply:
Ok, I sorted it out, you can recheck now.
I merged in your config name change and set the default to "False", this should be the old behavior.

Default is power pins are simply ignored all together (not copied but don't create mismatch errors either).

And with the flag on, it copies power from DEF to design.